### PR TITLE
Remove broken cookie on checkout script

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?

This PR removes the cookie with the name `[Object object]`, which affected the cart cookie on Checkout API.

#### What problem is this solving?

The cookie has invalid characters that the .NET Framework can't parse, and because of this the API can't read the cart cookie if this other cookie is present in the request headers.

#### How should this be manually tested?

N/A

#### Screenshots or example usage

N/A

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by jeffersontuc — APPROVED** [review]

File: src/checkout.coffee [hunk 1]
```diff
@@ -582,3 +582,6 @@ class Checkout
 window.vtexjs or= {}
 window.vtexjs.Checkout = Checkout
 window.vtexjs.checkout = new window.vtexjs.Checkout()
+
+if typeof document != 'undefined'
+	document.cookie = '[Object object]=; path=/; Max-Age=-99999999;'
```
Comment (lbebber) [inline src/checkout.coffee:587]: acho que a ordem das maiuscilas é diferente. nao é?
```suggestion
	document.cookie = '[object Object]=; path=/; Max-Age=-99999999;'
```

Comment (lucasecdb) [reply src/checkout.coffee:587]: ahh boa

File: src/checkout.coffee [hunk 2]
```diff
@@ -582,3 +582,6 @@ class Checkout
 window.vtexjs or= {}
 window.vtexjs.Checkout = Checkout
 window.vtexjs.checkout = new window.vtexjs.Checkout()
+
+if typeof document != 'undefined'
```
Comment (lbebber) [inline src/checkout.coffee:586]: talvez adiciona um comentario aqui falando da necessidade disso

[fiz assim](https://github.com/vtex/render-server/pull/747/files#diff-a4a59e076534b1a1ef73e7ced3cd82c936c984572e7c36bf3f107705904c66dfR639), ve o que acha

Comment (lucasecdb) [reply src/checkout.coffee:586]: boua, vou adicionar também

**Comment by lbebber** [comment]
aprovei, nao sei se vamos subir isso no fim das contas dado que o parceiro parece ter feito isso ja no codigo deles, mas se precisar ta aprovado

**Comment by lucasecdb** [comment]
closing, as the change was already made on the faulty script

**Pull request closed without merging**